x-pack/plugins/endpoint/public/embeddables/resolver/view/defs.tsx

```diff
@@ -0,0 +1,173 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { memo } from 'react';
+import { darken, saturate } from 'polished';
+
+import {
+  htmlIdGenerator,
+  // qualitative pallette functions.
+  //  See: https://elastic.github.io/eui/#/utilities/color-palettes
```

Comment:
The link says 'Qualitative palettes are best suited for communicating and comparing discrete data series.' Are you using them for that?

Reply:
That was sort of the idea... Nodes and the way they appear representing "categories" of data to display, but it would beggar the concept to say it is a 'discrete data series'.  I was more after the next part:  "for qualitative and _categorical_ data." We're torturing the colors anyway to get them to match the mocks - the aim here was just to provide some grounding with EUI-recommended colors (so e.g. if they change the colors returned by the palette function, they would have a way to matriculate here).  I could take it out - thoughts?